src/main/java/frc/robot/RobotContainer.java

```diff
@@ -60,6 +64,8 @@ public RobotContainer() {
   private void configureButtonBindings() {
     // camera button
     m_switchCameraButton = new JoystickButton(m_controller1, 1);
+    m_aimButton = new JoystickButton(m_flightStick, 12);
```

Comment:
Nit: Can we extract parameters into constants?

Reply:
its just the button number, so kind of wasteful.

Reply:
It would take 5 seconds, let's extract all the button definitions and motor definitions into its own static class so that we can provide a variable. This way, if we change the controller we only have to change one file, not everywhere a particular button is used

Reply:
done